chapters/6-analysis.tex

```diff
@@ -428,7 +428,7 @@ \subsection{Visualizing data}
 our team has developed a few resources to facilitate this workflow.
 First of all, we maintain easily-searchable data visualization libraries for both Stata\sidenote{
 	\url{https://worldbank.github.io/Stata-IE-Visual-Library}}, and R\sidenote{
-	\url{https://worldbank.github.io/r-econ-visual-library/}},.
+	\url{https://worldbank.github.io/r-econ-visual-library/}}.
```

Comment:
```suggestion
	\url{https://worldbank.github.io/r-econ-visual-library/}}
```

Reply:
I think this correctly places the period

Reply:
ok, but only when I changed the comma for the Stata reference for it to be consistent